### PR TITLE
pair programming- portfolio accesibility

### PR DESCRIPTION
**Pull request by Shri-sakthi-v**
### In this Portfolio project as a pair programming, we update the web accessibility in the page.
- [x] Page titles- Issues rectify.
- [x] Image text alternatives.
https://github.com/Shri-sakthi-v/mobile-first-portfolio/blob/7ebd75d9317a7bb43c5a041db055ff0f90016881/index.html#L56
- [x] Text headings-no issues found
- [x] Color contrast- no issues found.
- [x] Resize-no issues found
- [x] Interaction - https://github.com/Shri-sakthi-v/mobile-first-portfolio/blob/fa06b382230d18315d0244f5293326b111524e7e/index.html#L243-L245
- [x] Moving content-no issues found
- [x] Multimedia-no issues found
- [x] The basic structure of the page-no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -241,8 +241,8 @@ <h1 class="contact-title">Contact Me</h1>
                 </p>
                 <form id="form_id" action="https://formspree.io/f/mlekbobg" method="post" class="contact">
                         <label for="name"><input type="text" id="name" name="name" placeholder="enter your name" required maxlength="30"></label>
-                        <input type="email" id="email" name="email" placeholder="enter your email" required>
-                        <textarea name="message" id="" cols="30" rows="10" required placeholder="Enter the message" maxlength="500"></textarea>
+                        <label for="email"><input type="email" id="email" name="email" placeholder="enter your email" required></label>
+                        <textarea for="textarea" name="message" id="" cols="30" rows="10" required placeholder="Enter the message" maxlength="500"></textarea>
```
Comment (Shri-sakthi-v) [inline index.html:245]: #8

File: index.html [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
     <link rel="stylesheet" type="text/css" href="style.css">
-    <title>Portfolio</title>
+    <title>Shri Sakthi Portfolio Website</title>
 </head>
 <body>
 
     <div class="starting-page"> 
         <header >
             <nav id="navigator_bar">
                 <a class="logo" href="#"><h2>Mylogo</h2></a>
```
Comment (souad988) [inline index.html:19]: Good job so far @Shri-sakthi-v , Just a small thing to mention, the color of the logo doesn't have enough contrast with the background kindly make it a bit darker for a better user experience

File: index.html [hunk 3]
```diff
@@ -181,15 +181,15 @@ <h2>LETS CONNECT</h2>
                 </div>
```
Comment (souad988) [inline index.html:181]: Kindly use a hx tag for the title of this block as it is a headline , I mean line 176  ` <p>Languages</p>`

File: index.html [hunk 4]
```diff
@@ -240,9 +240,9 @@ <h1 class="contact-title">Contact Me</h1>
                      that needs coding.I’d love to help with it 
                 </p>
                 <form id="form_id" action="https://formspree.io/f/mlekbobg" method="post" class="contact">
-                        <input type="text" id="name" name="name" placeholder="enter your name" required maxlength="30">
-                        <input type="email" id="email" name="email" placeholder="enter your email" required>
-                        <textarea name="message" id="" cols="30" rows="10" required placeholder="Enter the message" maxlength="500"></textarea>
+                        <label for="name"><input type="text" id="name" name="name" placeholder="enter your name" required maxlength="30"></label>
+                        <label for="email"><input type="email" id="email" name="email" placeholder="enter your email" required></label>
+                        <textarea for="textarea" name="message" id="" cols="30" rows="10" required placeholder="Enter the message" maxlength="500"></textarea>
                     <button type="submit" name="submit">Get in touch</button>
```
Comment (souad988) [inline index.html:246]: This button doesn't have the same colors as in figma template and this causes contrast issues 
Please refer to figma and make these changes also for hover effect 

get in touch contrast

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Shri-sakthi-v , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔Good use of github workflow
✔Good PR title
✔Fair contributions between peers

## Required Changes ♻️
_Check the comments under the review._
_Also please make sure that your PR summary is short , you don't need to list changes done in each line just headlines about the overall work 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Shri-sakthi-v) [reply index.html:246]: Thank you for notifying the correction. issue been resolved

File: index.html (hunk 3 above)
Comment (Shri-sakthi-v) [reply index.html:181]: Thank you for notifying the correction. the issue has been resolved

**Review by levy002 — APPROVED** [review]
## Hello @Shri-sakthi-v 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**